otter/integration/tests/test_lbsh.py

```diff
@@ -70,24 +70,23 @@ def test_oob_deleted_clb_node(self):
         clb = self.helper.clbs[0]
 
         nodes = yield clb.list_nodes(self.rcs)
-        self.assertEqual(len(nodes['nodes']), 0,
-                         "There should be no nodes on the CLB yet.")
+        yield clb.wait_for_nodes(
+            self.rcs, HasLength(0), timeout=timeout_default)
 
         group, _ = self.helper.create_group(min_entities=1)
         yield self.helper.start_group_and_wait(group, self.rcs)
 
+        yield clb.wait_for_nodes(
+            self.rcs, HasLength(1), timeout=timeout_default)
+
         nodes = yield clb.list_nodes(self.rcs)
-        self.assertEqual(
-            len(nodes['nodes']), 1,
-            "There should be 1 node on the CLB now that the group is active.")
         the_node = nodes["nodes"][0]
 
         yield clb.delete_nodes(self.rcs, [the_node['id']])
 
         nodes = yield clb.list_nodes(self.rcs)
```

Comment:
This line can probably removed since we are just waiting for the nodes to be 0  :)